x/ecocredit/client/basket/tx.go

```diff
@@ -113,9 +113,9 @@ Flags:
 				return err
 			}
 
-			if minStartDateString == "" && startDateWindowString == "" {
+			if minStartDateString == "" && startDateWindow == 0 {
```

Comment:
is `DateCriteria` actually required? We are allowing empty `DateCriteria` in [ValidateBasic](https://github.com/regen-network/regen-ledger/blob/release/v2.2.x/x/ecocredit/basket/msg_create.go#L88-L90).

Reply:
not, it's not required.